### ISSUE TITLE
2.16 macOS users are seeing the “Updates available” modal on Stage

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/mozilla-vpn-client#8407]
**Affected versions:**

* 2.16.0

**Tested Platforms:**

* macOS;

**Prerequisites:**

* Have Mozilla VPN app **2.16** installed and open;
* Have a Mozilla account with an active subscription;
* Be signed in - **Stage server**;  

**Steps to reproduce:**

1. Go to Settings - About us screen and press the “Check for updates” button;
1. Observe the modal displayed;

**Expected result:**

* “You are up to date” modal is displayed;

**Actual result:**

* “Updates available” modal is displayed;

**Notes:**

* Due to the crash discovered in 2.16 for macOS M1 users, the update should not be visible.

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/VPN-5774)
┆Reporter: Bianca Hidecuti

**Comment by data-sync-user** [comment]
➤ Bianca Hidecuti commented:

Verified this as fixed on Mozilla VPN 2.16.0, using macOS 12.6.6/14.0, and 2.16 users are now getting the “You are up to date” modal, while on Stage server. 

However, while doing some additional checks in Production, we found out that currently, macOS users are not seeing updates available while using Mozilla VPN ≤ 2.15.1. Seems like the update was stopped for all macOS users.

Will leave this “In Progress” until we get a confirmation for the behavior from the Production environment. 

For more context, see https://mozilla.slack.com/archives/C01DCUKG95E/p1698330021890559 ( https://mozilla.slack.com/archives/C01DCUKG95E/p1698330021890559|smart-link ) 

cc: Beatriz Rizental Machado / Lesley Norton
